Dockerfile

```diff
@@ -24,4 +33,15 @@ RUN set -e \
  && SECRET_KEY=temp HOOVER_DB='postgresql://search:search@search-pg:5432/search' ./manage.py collectstatic --noinput \
  && chmod +x /wait
 
-CMD ./runserver
+RUN chown -R $UID:$GID /opt/hoover/search
+
+ENV USER_NAME $USER_NAME
+ENV DATA_DIR /opt/hoover/search
```

Comment:
let's not set up this variable, as it's not used anywhere.

also, there's no DATA_DIR in this repo, since all it needs is the posgresql for data